package.json

```diff
@@ -10,7 +10,7 @@
   "author": "Andrew Reid <glyph@mycelial.technology>",
   "license": "AGPL-3.0",
   "dependencies": {
-    "ssb-validate2-rsjs-node": "~0.3.0",
+    "ssb-validate2-rsjs-node": "~0.4.0",
     "ssb-validate2-rsjs-wasm": "~0.3.0"
```

Comment:
`-wasm` is now also at version `0.4.0`

Reply:
It has to be published on npm, though, I'll do that 1st

Reply:
Ah of course. Thank you 🙏🏻